File: VB/Customer.vb
Namespace TestGridLookUpEditCSV

    Public Class Customer

        Public Sub New()
            ID = 0
            Name = ""
            Info = ""
        End Sub

        Public Sub New(ByVal id As Integer, ByVal name As String, ByVal info As String)
            Me.ID = id
            Me.Name = name
            Me.Info = info
        End Sub

        Public Property ID As Integer

        Public Property Name As String

        Public Property Info As String
    End Class
End Namespace

File: CS/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TestGridLookUpEditCSV {
    public class Customer {
        public Customer() {
            this.ID = 0;
            this.Name = "";
            this.Info = "";
        }

        public Customer(int id, string name, string info) {
            this.ID = id;
            this.Name = name;
            this.Info = info;
        }

        public int ID { get; set; }
        public string Name { get; set; }
        public string Info { get; set; }
    }
}
